pontoon/base/models.py

```diff
@@ -472,6 +472,25 @@ def get_top_instances(self):
 @python_2_unicode_compatible
 class Locale(AggregatedStats):
     code = models.CharField(max_length=20, unique=True)
+
+    # Codes related to Microsoft products.
+    ms_translator_code = models.CharField(
+        max_length=20,
+        blank=True,
+        help_text="""
+        Microsoft Translator maintains its own list of <a href="https://msdn.microsoft.com/en-us/library/hh456380.aspx">supported locales</a>.
+        Choose a locale from that list that's is the closest match or leave it blank to disable support for this feature.
+        """
+    )
+    ms_terminology_code = models.CharField(
+        max_length=20,
+        blank=True,
+        help_text="""
+        Microsoft Terminology uses locale codes standarized by ISO-639. Add a code of compliant locale or leave blank to disable
```

Comment:
*Microsoft Terminology uses locale codes standarized by ISO-639.* -> that's also the case for Pontoon. The difference is that in Microsoft Terminology [language codes currently include both the language and the country/region](https://www.microsoft.com/Language/en-US/Microsoft-Terminology-API.aspx).

I suggest you change the help text to:
*Microsoft Terminology uses language codes that include both the language and the country/region. Chose a matching locale from <a href="https://www.microsoft.com/Language/en-US/Search.aspx">the list</a> or leave blank to disable support for Microsoft terminology.*